### PR TITLE
Print plugin variables value's when executing appbuilder plugin

### PR DESCRIPTION
**Pull request by Fatme**
Fixes http://teampulse.telerik.com/view#item/281723

### CODE REVIEW

File: lib/plugins-data.ts [hunk 1]
```diff
@@ -13,56 +14,88 @@ export class CordovaPluginData implements IPlugin {
...
+		return result;
+	}
+
+	private getBasicPluginInformation(): string[] {
+		var nameRow = this.buildRow("Plugin", this.data.Name);
+		var identifierRow = this.buildRow("Identifier", this.data.Identifier);
+		var versionRow = this.buildRow("Version", this.data.Version);
+		var urlRow = this.buildRow("Url", this.data.Url);
+
+		 var result = [nameRow, identifierRow, versionRow, urlRow];
```
Comment (rosen-vladimirov) [inline lib/plugins-data.ts]: space befor var (don't hate me too much)

File: lib/plugins-data.ts [hunk 2]
```diff
@@ -13,56 +14,88 @@ export class CordovaPluginData implements IPlugin {
...
+		return util.format("    %s: %s", key, value);
+	}
+
+	protected composePluginInformation(additionalPluginData: string[]) 	{
+		var result = this.getBasicPluginInformation();
+
+		result = result.concat(additionalPluginData);
+		result = result.concat(this.getPluginVariablesInfo());
+
+		return result;
```
Comment (teobugslayer) [inline lib/plugins-data.ts]: maybe ``` return _.flatten(this.getBasicPluginInformation(), additionalPluginData, this.getPluginVariablesInfo())```?

File: lib/plugins-data.ts [hunk 3]
```diff
@@ -13,56 +14,88 @@ export class CordovaPluginData implements IPlugin {
...
+				var variables = pluginVariablesData[this.data.Identifier];
+				var variableNames = _.keys(variables);
+				if(variableNames.length > 0) {
+					var output:string[] = [];
+					output.push(util.format("    Variables for %s configuration: %s", configuration, os.EOL));
+					_.each(variableNames, (variableName:string) => {
+						output.push(util.format("        %s: %s %s", variableName, variables[variableName], os.EOL));
+					});
+
+					result.push(output.join(""));
```
Comment (teobugslayer) [inline lib/plugins-data.ts]: Why not ```output.join(os.EOL)``` and remove it from the other places?

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**